stories/Form Elements/Button.stories.tsx

```diff
@@ -64,26 +64,37 @@ export const Primary: Story = {
 };
 export const Secondary: Story = { args: { secondary: true, children: 'Secondary' } };
 export const Reverse: Story = { args: { reverse: true, children: 'Reverse' } };
+
+/**
+ * 
+ * Disabled buttons have poor contrast and can confuse some users. Only use them if user research shows it makes things easier for users to understand.
+ * 
+ */
+
 export const Disabled: Story = { args: { disabled: true, children: 'Disabled' } };
 export const LinkButton: Story = { args: { href: '/', children: 'As a Link' } };
 export const ForceButton: Story = { args: { as: 'button', children: 'As a Button' } };
 export const ForceAnchor: Story = { args: { as: 'a', children: 'As an Anchor' } };
-export const DebouncedButton: Story = {
+
+/**
+ * You can test this button by opening the browser console. It will log the current dateTime once per debounce.
+ * 
+ * Debounced buttons can optionally provide `debounceTimeout` (measured in miliseconds) to adjust the length of the debounce.
```

Comment:
Nitpick - typo for milliseconds
```suggestion
 * Debounced buttons can optionally provide `debounceTimeout` (measured in milliseconds) to adjust the length of the debounce.
```